src/app/fdctl/monitor/monitor.seccomppolicy

```diff
@@ -0,0 +1,9 @@
+unsigned int drain_output_fd
+write: (or (eq (arg 0) 1) 
```

Comment:
Did you add comments? Can you please copy the comments over for these why each is needed etc

Reply:
I did add comments. They are supported. Let me do a round.